src/components/30-organisms/modal/index.scss

```diff
@@ -0,0 +1,90 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
Those rules need to be removed (html/boduy). This is dangerous. And it seems that it's not needed.